src/components/Admin/AddPopUpClinic/AdminPopUpClinic.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import { Card, Form, FormLayout, TextField } from "@shopify/polaris";
+import { useTranslation } from "react-i18next";
+import { postalCodeIsValid } from "../../../utils/postalCode";
+import { phoneNumberIsValid } from "../../../utils/phoneNumber";
+
+export interface IPopUpClinic {
+  name: string;
+  date: string;
+  street: string;
+  city: string;
+  province: string;
+  postalCode: string;
+  phone: string;
+  websiteUrl: string;
+  [key: string]: string;
```

Comment:
I don't think you need to index this object by arbitrary keys, you only want to index by known keys.